### DISCUSSION TITLE
More rate of GPU utilization instead of one gpu for one trial

### DISCUSSION THREAD

**Discussion by rickyyyxu** [microsoft/nni#4288] [Q&A] [answered]
Hello, in nni, I can set `trialGpuNumber: 1`  to let one trial use a gpu. If I have two gpu, just two trial can concurrently run.
So, how can one gpu support many trials in the same experiment when gpu space is enough?

Thank you~

**Comment by QuanluZhang** [comment, marked as answer]
@coasxu , you can use `maxTrialNumberPerGpu`, more detailed here: https://nni.readthedocs.io/en/stable/reference/experiment_config.html#maxtrialnumberpergpu

**Comment by rickyyyxu** [comment]
Wow, it's really helpful, I will get a try!
